src/Rule/AtLeast.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Validator\Rule;
+
+use Yiisoft\Validator\Rule;
+use Yiisoft\Validator\Result;
+use Yiisoft\Validator\DataSetInterface;
+
+/**
+ * AtLeastValidator checks if at least $min of many attributes are filled.
+ */
+class AtLeast extends Rule
+{
+    /**
+     * The minimum required quantity of filled attributes to pass the validation.
+     * Defaults to 1.
+     */
+    private int $min = 1;
+
+    /**
+     * The list of required attributes that will be checked.
+     */
+    private array $attributes = [];
+
+    /**
+     * The list of alternative required attributes that will be checked.
+     */
+    private array $alternativeAttributes = [];
+
+    /**
+     * Message to display in case of error
+     */
+    private string $message = 'The model is not valid. Must have at least "{min}" filled attributes.';
+
+    public function __construct(array $data)
```

Comment:
That's not how the rest of validators are configured. Please use the same style.

Reply:
I updated the PR, is it ok?.